app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,27 @@
+class CommentsController < ApplicationController
+  def create
+    @comment = Comment.create!(comment_params)
```

Comment:
Suggest not using `!` methods in controllers, as they lead to a generic error page if the request fails. Instead, using an `if` statement (like we did in the validations lesson) with either a redirect + flash message or re-rendering the form w/ error messages leads to a nicer UX.